helpers/withInstance.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react'
+
+const withInstance = client => {
```

Comment:
Since this is a hook, and the official guideline for hooks is to use names starting with `use`, I would rename this to `useInstance`.